node/src/BaseClient.ts

```diff
@@ -266,7 +267,7 @@ export type GlideString = string | Buffer;
 /**
  * Enum representing the different types of decoders.
  */
-export const enum Decoder {
+export enum Decoder {
```

Comment:
This is needed to use `Decoder` in `DecoderOption` type.
According to [docs](https://dev.to/muszynov/something-about-typescript-enums-56jp#:~:text=const%20enums&text=When%20you%20use%20a%20const,in%20the%20compiled%20JavaScript%20output.):
> Const enums cannot be used in a context where a type is expected, such as in a function signature or type annotation.